src/smart-components/group/role/group-roles.js

```diff
@@ -48,15 +48,17 @@ const generateOuiaID = (name) => {
   return name.toLowerCase().includes('default access') ? 'dag-add-role-button' : 'add-role-button';
 };
 
-const addRoleButton = (isDisabled, ouiaId) => {
+const addRoleButton = (isDisabled, ouiaId, customTooltipText) => {
   const addRoleButtonContent = (
     <Button ouiaId={ouiaId} variant="primary" className="rbac-m-hide-on-sm" aria-label="Add role" isAriaDisabled={isDisabled}>
       Add role
     </Button>
   );
 
   return isDisabled ? (
-    <Tooltip content="All available roles have already been added to the group">{addRoleButtonContent}</Tooltip>
+    <Tooltip content={customTooltipText ? customTooltipText : 'All available roles have already been added to the group'}>
```

Comment:
No need for ternary operator
```suggestion
    <Tooltip content={customTooltipText || 'All available roles have already been added to the group'}>
```